nextflow_schema.json

```diff
@@ -570,7 +570,7 @@
                     "description": "Maximum amount of time that can be requested for any single job.",
                     "default": "240.h",
                     "fa_icon": "far fa-clock",
-                    "pattern": "^[\\d\\.]+\\.*(s|m|h|d)$",
+                    "pattern": "^(\\d+\\.?\\s*(s|m|h|day)\\s*)+$",
```

Comment:
Sorry, what is the context for this change?

Reply:
My bad...these were ongoing changes that I introduced dynamically as I was spotting, updating and changing things in the rnaseq pipeline. See https://github.com/nf-core/tools/issues/858